### DISCUSSION TITLE
Unable to sign up with email while deploying supabase with docker

### DISCUSSION THREAD

**Discussion by DingGao1124** [supabase/supabase#27068] [Questions] [answered]
when i click the link in my email, the link of "Accept the invite" is shown as below:
![image](https://github.com/supabase/supabase/assets/136151131/619c46c6-39a8-4833-98d1-07ea44cb9847)
![image](https://github.com/supabase/supabase/assets/136151131/2329e051-1a80-4640-9679-320eadccc975)
i need to fix the "kong" to "127.0.0.1:8000" manually to take it into effect, can i config it previouly, just like the params in .env?
![image](https://github.com/supabase/supabase/assets/136151131/8ecd432a-a0cf-445b-99d3-70cce7f69a0f)

**Comment by KRQQN** [comment]
Did you manage to fix this? :)

**Reply by DingGao1124** [reply]
not yet :(

**Comment by KRQQN** [comment, marked as answer]
This fixed it for me:
https://github.com/orgs/supabase/discussions/17961#discussioncomment-8287569

to be clear: prefixing the auth/v1/verify with localhost.
E.g : `MAILER_URLPATHS_CONFIRMATION="http:/localhost:8000/auth/v1/verify`

**Reply by DingGao1124** [reply]
Thank you very much, the problem has gone!
